WordPress/src/main/java/org/wordpress/android/ui/main/emailverificationbanner/EmailVerificationBanner.kt

```diff
@@ -0,0 +1,321 @@
+                fontWeight = FontWeight.Bold,
+                color = MaterialTheme.colorScheme.onSurface
+            )
+        }
+
+        Text(
+            text = stringResource(R.string.me_email_verification_verify_email_description),
+            style = MaterialTheme.typography.bodyLarge,
+            modifier = Modifier.padding(top = 8.dp)
+        )
+
+        Text(
+            text = stringResource(R.string.me_email_verification_verify_email_send_link),
+            style = MaterialTheme.typography.bodyLarge,
+            color = colorResource(R.color.jetpack_green_50),
+            modifier = Modifier
+                .padding(top = 8.dp)
+                .clickable {
+                    onSendLinkClick()
+                }
+        )
+    }
+}
+
+/**
+ * Banner when user's email hasn't been verified but a verification link has been requested
+ */
+@Composable
+private fun EmailVerificationSendingBanner() {
+    EmailVerificationContainer {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            CircularProgressIndicator(
+                strokeWidth = 2.dp,
+                modifier = Modifier
+                    .size(24.dp)
+            )
+            Text(
+                text = stringResource(R.string.me_email_verification_sending),
+                style = MaterialTheme.typography.bodyLarge,
+                fontWeight = FontWeight.Bold,
+                color = MaterialTheme.colorScheme.onSurface,
+                modifier = Modifier.padding(start = 16.dp)
+            )
+        }
+
+        Row {
+            Text(
+                text = stringResource(R.string.me_email_verification_sending_description),
+                style = MaterialTheme.typography.bodyLarge,
+                modifier = Modifier.padding(top = 8.dp)
+            )
+        }
+    }
+}
+
+/**
+ * Banner when user's email hasn't yet been verified but a verification link has been sent
+ */
+@Composable
+private fun EmailVerificationSentBanner(
+    emailAddress: String,
+    onResendLinkClick: () -> Unit,
+) {
+    EmailVerificationContainer {
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                painter = painterResource(R.drawable.ic_gridicons_checkmark_circle),
+                contentDescription = null,
+                tint = colorResource(R.color.jetpack_green_50),
+                modifier = Modifier.padding(end = 8.dp)
+            )
+            Text(
+                text = stringResource(R.string.me_email_verification_sent),
+                style = MaterialTheme.typography.bodyLarge,
+                fontWeight = FontWeight.Bold,
+                color = colorResource(R.color.jetpack_green_50)
+            )
+        }
+
+        Text(
+            text = stringResource(
+                R.string.me_email_verification_sent_description,
+                emailAddress
+            ),
```

Comment:
I noticed in the video that the full text was "Check your inbox at for the confirmation ...". Maybe the `emailAddress` can be an empty string? In that case, we should not use the string format `... at %s ...`.